docs/roadmap.md

```diff
@@ -4,6 +4,149 @@ title: Babel Roadmap
+
+This is far from being a complete list of all the new features or important changes that we'll bring to Babel, but it's a good summary if you are interested in the general direction that the project is moving to. We might actually not do every listed point, or we might delay some of them to the next year. Some of them have clear starting and ending points, while others need more research or [RFCs](https://github.com/babel/rfcs).
+
+If your company is interested in some of these and would like to directly sponsor their implementation please [reach out](mail:team@babeljs.io)!
+
+## Babel 2021 Roadmap
+
+### Babel 8
+
+We have been talking about the Babel 8 release for more than one year (actually, we initially scheduled it for about one year ago)! However, we are now closer then ever to actually release it.
+
+Most of the remaining tasks are in the [tracking issue](https://github.com/babel/babel/issues/10746), but there are still a few blockers:
+- We want to drop support for [Node.js 10](https://github.com/nodejs/Release), which stops being maintained on 2021-04-30;
+- We would like to release Babel as a pure ESM package. We are now in the process of converting our sources to be compatible with Node.js' ESM implementation, and while doing it we are examining how we can make it easier for people that currently use Babel to compile ESM to CJS;
+- We are trying to align our TypeScript AST with the `typescript-eslint` project. Our ASTs are _almost_ identical, but we need to introduce some small breaking changes to fully align;
+- Our release infrastructure doesn't support yet pre-releases, or using multiple "main" branches (one for Babel 8 and one for Babel 7);
+- We haven't figured out yet a policy for Babel 7 maintainance.
+
+### Implement new TC39 proposals
+
+Babel can currently parse all the Stage 3 proposals, and we can transform all of them except for top-level await, import assertions and JSON modules (which are best handled by bundlers working with the dependencies graph).
+
+We support all the Stage 2 proposals except for:
+- The new iteration of the decorators proposal (we need to implement both parsing and transform);
+- Transform for the Module Blocks proposal (we implemented parsing in Babel 7.13.0).
+
+We will implement support for decorators, and investigate if and how we can implement transform for module blocks.
+
+While we don't support many Stage 1 proposals, there have been recent updates to the pipeline operator and to do expressions: since we already support those proposals and the community is quite excited about them, we will update our implementations.
+
+There are also other proposal, such as pattern matching, that we didn't implement yet because their champions expect to do significant changes to the syntax and semantics. However, we are closely following their development to implement them in Babel as soon as they stabilize a bit.
```

Comment:
```suggestion
There are also other proposals, such as pattern matching, that we have not yet implement because their champions expect significant changes to the syntax and semantics. However, we are closely following their development, and will implement them in Babel as soon as they stabilize a bit.
```